capabilities/learning-culture.md

```diff
@@ -1,85 +1,108 @@
 
 ## Assessment
 
-To assess how mature your team or organization is in this capability, answer the question below.
+To assess how mature your team or organization is in this capability, complete this short exercise.
 
-Don't worry if the description doesn't exactly match your situation. The descriptions are meant as examples of situations that would qualify for the associated score. Generally, score yourself a 1 if this capability is completely missing from your team, a 2 if there is a lot of room for improvement, a 3 if there is some room for improvement, and a 4 if your team is exemplary in this capability.
+Consider the descriptions below and score your team on the Learning Culture capability. Generally, score a 1 if learning is limited to static knowledge, a 2 if your team demonstrates ad hoc learning but you feel there is a LOT of room for improvement, a 3 if learning is encouraged and you feel there is some room for improvement, and a 4 if learning is actively prioritized and you feel your team is exemplary in the area of Learning Culture.
+
+Don't worry if the description doesn't exactly match your situation. These descriptions are meant to be examples of situations that would qualify for the associated score.
 
 1. **Static Knowledge:** Learning is limited to onboarding and initial training, with little emphasis on ongoing development or skill-building.
 2. **Ad Hoc Learning:** Teams occasionally attend conferences or workshops, but learning is not a prioritized or structured part of the organization's culture.
 3. **Encouraged Learning:** Learning is valued and encouraged, with some resources and opportunities provided for professional development, but it's not a core part of the organization's identity.
-4. **Learning As A Competitive Advantage:** Learning is deeply ingrained in the organization's culture, viewed as a key driver of improvement and innovation, and actively prioritized and invested in to stay ahead of the curve.
+4. **Learning as a Competitive Advantage:** Learning is deeply ingrained in the organization's culture, viewed as a key driver of improvement and innovation. It is actively prioritized and invested in, helping the team to stay ahead of the curve.
+
+The number you selected represents your overall score for this capability. If you feel like your team or organization fits somewhere in between two scores, it's okay to use a decimal. For example, if you think your team's learning represents something between static knowledge and ad hoc learning, then you would score a 2.5.
+
+Generally, an overall score below 3 means you'll likely gain a lot of value from experimenting with some of the supporting practices listed below. An overall score higher than 3 generally means you and your team are largely proficient, or well on your way to becoming proficient, in the area of Learning Culture; you would likely benefit from evaluating your scores in other capabilities.
```

Comment:
Edited here to below/higher than 3, per our conversation in Test Automation capability.